openc3/lib/openc3/interfaces/file_interface.rb

```diff
@@ -72,15 +72,14 @@ def initialize(
     end
 
     def connect
-      super()
-
       if @telemetry_read_folder
         @listener = Listen.to(@telemetry_read_folder, force_polling: @polling) do |modified, added, removed|
           @queue << added if added
         end
         @listener.start # starts a listener thread--does not block
       end
 
+      super()
```

Comment:
Sending the command in the `connect` method only works if we do the `super()` last.